src/main/scala/is/hail/variant/Variant.scala

```diff
@@ -132,7 +132,10 @@ object Variant {
     val r = a.asInstanceOf[Row]
     val l = r.getAs[Locus](0)
     val alleles = r.getAs[IndexedSeq[String]](1)
-    Variant(l.contig, l.position, alleles(0), alleles.tail.map(x => AltAllele(alleles(0), x)))
+    if (l == null || alleles == null)
```

Comment:
I think this is OK, but I'm wondering, why were the locus/alleles null?  That's quite strange in the current code.

Reply:
The bug ended up being further upstream, but Tim says that the keys are allowed to be null and since minrep calls this outside of the tests, I left it in.